### ISSUE TITLE
XOR?   (Priority: "Nice to have")

### ISSUE THREAD

**Issue by bobbyjim** [MEGA65/mega-basic64#4]
Commodore BASIC didn't have an XOR.  Is there room to add one?

**Comment by gardners** [comment]
Yes, higher versions of C= BASIC have it.  Note that work on this is largely suspended while we instead direct effort onto our OpenROMs project (github.com/mega65/open-roms).

Paul.

**Comment by bobbyjim** [comment]
Ah yes, I understand.  I hope it's still okay to use this as a backlog in the meantime.

**Comment by gardners** [comment]
Sure. But I'd also add it to the issues on the open-roms repo, as that is more likely to be finished first, and may in fact completely displace this one.

**Comment by bobbyjim** [comment]
D'oh.  OK!
